packages/dialect-react-ui/entities/notifications/Notification.tsx

```diff
@@ -0,0 +1,78 @@
+import clsx from 'clsx';
+import type { PublicKey } from '@solana/web3.js';
+import { LinkifiedText } from '../../components/common';
+import { Img, P } from '../../components/common/preflighted';
+import { useTheme } from '../../components/common/providers/DialectThemeProvider';
+import { useIdentity } from '@dialectlabs/react-sdk';
+import { shortenAddress } from '../../utils/displayUtils';
+
+type Props = {
+  author?: PublicKey;
+  message: string;
+  timestamp: number | Date;
+};
+
+const timeFormatter = new Intl.DateTimeFormat('en-US', {
```

Comment:
We have many time formatters, is it possible to unify them somehow, or at least put them into 1 place and describe the best we can?

Reply:
this is the only one formatter for notifications actually
we have the other formatter for chat, but it has a bit different business logic

Reply:
Yeah, but they are still part of the same package. I would try to put them in the same place if possible, like `shared/utils` or something

Reply:
I don't think it can be reused somewhere, it's super related to this particular component visualization